LibTest/js_interop/JSExport/jsExport_A06_t03.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Only concrete instance members can and will be wrapped, and it's
+/// an error to annotate other members with this annotation.
+///
+/// @description Checks that it is a compile-time error if a global variables,
+/// functions, getters or setters are annotated with `JSExport()`.
+/// @author sgrekhov22@gmail.com
+
+import 'dart:js_interop';
+
+@JSExport()
+int variable = 42;
+//  ^^^^^^^^
+// [analyzer] unspecified
+// [web] unspecified
+
+@JSExport()
+void func() {}
+//   ^^^^
```

Comment:
More of a meta-level question, once these get rolled into the SDK, are we skipping them and linking to the issue to implement them next to the skip?

It would be nice to add these warnings/errors for unrelated declarations, but I don't think I'll get to it soon.